shared/django_apps/dummy_settings.py

```diff
@@ -21,10 +28,29 @@
     "shared.django_apps.reports",
 ]
 
-MIDDLEWARE = []
-
-TEMPLATES = []
+# Needed for makemigrations to work
```

Comment:
Had to bring these 2 back since the `makemigrations` command complained otherwise